node/typedoc.json

```diff
@@ -1,8 +1,7 @@
 {
     "cleanOutputDir": true,
-    "entryPoints": [
-        "src/index.ts"
-    ],
+    "entryPoints": ["src/index.ts"],
     "out": "apidocs",
+    "readme": "src/README.md",
```

Comment:
This is the only functional change. The rest just updates to current dev dependency versions.